ppr-ui/src/components/mhrRegistration/HomeOwners/AddHomeOwnerOrganization.vue

```diff
@@ -0,0 +1,227 @@
+            <v-col>
+              <p>
+                You can look-up a B.C. business by entering the name of the
+                business or the incorporation number (including Societies and
+                extra-provincial companies registered in B.C.). If the name of
+                the organization does not appear in the look-up, enter the full
+                legal name of the organization.
+              </p>
+              <p>
+                <v-expansion-panels>
+                  <v-expansion-panel>
+                    <v-expansion-panel-header
+                      class="pa-0 primary--text"
+                      :hide-actions="true"
+                    >
+                      <div>
+                        <v-icon color="primary" v-on="on"
+                          >mdi-information-outline
+                        </v-icon>
+                        Help with Proprietorships / Partnerships
+                      </div>
+                    </v-expansion-panel-header>
+                    <v-expansion-panel-content class="help-toggle-content">
+                      Registered owners of a manufactured home cannot be a sole
+                      proprietorship, partnership or limited partnership. The
+                      home must be registered in the name of the proprietor or
+                      partner (person or organization).
+                    </v-expansion-panel-content>
+                  </v-expansion-panel>
+                </v-expansion-panels>
+              </p>
+            </v-col>
+          </v-row>
+          <v-row>
+            <v-col>
+              <v-text-field
+                filled
+                id="org-name"
+                label="Full legal name of the organization"
+                v-model="searchValue"
+                :hide-details="hideDetails"
+              />
+              <auto-complete
+                :searchValue="autoCompleteSearchValue"
+                :setAutoCompleteIsActive="autoCompleteIsActive"
+                v-click-outside="setCloseAutoComplete"
+                @search-value="setSearchValue"
+              >
+              </auto-complete>
+            </v-col>
+          </v-row>
+          <label class="generic-label" for="manufacturer-name">
+            Phone Number
+          </label>
+          <v-row>
+            <v-col cols="6">
+              <v-text-field
+                id="phone-number"
+                filled
+                :rules="maxLength(15)"
+                label="Phone Number"
+                data-test-id="phone-number"
+              />
+            </v-col>
+            <v-col cols="6">
+              <v-text-field
+                id="phone-ext"
+                filled
+                :rules="maxLength(15)"
+                label="Extension (Optional)"
+                data-test-id="phone-ext"
+              />
+            </v-col>
+          </v-row>
+
+          <label class="generic-label" for="manufacturer-name">
```

Comment:
Is this intended to be `manufacturers-name`?

Reply:
Ooops, a typo, should label for mailing address. Will fix.

Reply:
Updated